features/git-extract/multiple_commits/with_remote_origin/with_open_changes.feature

```diff
@@ -32,11 +32,20 @@ Feature: git extract: extracting multiple commits (with open changes)
     And I end up on the "refactor" branch
     And I still have my uncommitted file
     And I have the following commits
-      | BRANCH   | LOCATION         | MESSAGE            | FILE NAME        |
-      | main     | local and remote | remote main commit | remote_main_file |
-      | feature  | local            | feature commit     | feature_file     |
-      |          |                  | refactor1 commit   | refactor1_file   |
-      |          |                  | refactor2 commit   | refactor2_file   |
-      | refactor | local and remote | remote main commit | remote_main_file |
-      |          |                  | refactor1 commit   | refactor1_file   |
-      |          |                  | refactor2 commit   | refactor2_file   |
+      | BRANCH   | LOCATION         | MESSAGE            |
+      | main     | local and remote | remote main commit |
+      | feature  | local            | feature commit     |
+      |          |                  | refactor1 commit   |
+      |          |                  | refactor2 commit   |
+      | refactor | local and remote | remote main commit |
+      |          |                  | refactor1 commit   |
+      |          |                  | refactor2 commit   |
+    And now I have the following committed files
+      | BRANCH   | NAME             |
+      | main     | remote_main_file |
+      | feature  | feature_file     |
+      | feature  | refactor1_file   |
+      | feature  | refactor2_file   |
+      | refactor | refactor1_file   |
+      | refactor | refactor2_file   |
+      | refactor | remote_main_file |
```

Comment:
But if each commit is dealing with a different file then I don't see any reason to inspect the files. I'll spike out an example and we can discuss there.